### PR TITLE
fix(navbar): add separator for top single links

### PR DESCRIPTION
**Pull request by dfvalero**
Before (check Save link):

Screenshot 2020-06-16 at 11 40 12

After:

Screenshot 2020-06-16 at 11 41 24

### CODE REVIEW

File: src/components/organisms/Navbar/NavbarLinksList/NavbarLinksList.tsx [hunk 1]
```diff
@@ -1,13 +1,35 @@
-import React, { Fragment } from 'react';
+import React from 'react';
 
 import NavbarDropdown, { ItemProps } from '../NavbarDropdown/NavbarDropdown';
 import { NavigationItem, NavbarLinksListProps } from '../Navbar/Navbar';
+import styled from 'styled-components';
+import { colors, spacing } from '../../../../constants';
+import { minMedia } from '../../../../helpers/responsiveness';
```
Comment (admmasters) [inline src/components/organisms/Navbar/NavbarLinksList/NavbarLinksList.tsx:7]: Any plans to use some kind of aliasing - these paths are a bit crazy lol.

Comment (dfvalero) [reply src/components/organisms/Navbar/NavbarLinksList/NavbarLinksList.tsx:7]: I will open an issue right now!

Comment (dfvalero) [reply src/components/organisms/Navbar/NavbarLinksList/NavbarLinksList.tsx:7]: https://github.com/zopaUK/react-components/issues/508

**Pull request merged**